src/Tracks/SpotifyPlaylist.php

```diff
@@ -28,7 +28,7 @@ public function __construct(string $uri)
     {
         # If this is a spotify playlist ID and not a URI then convert it to a URI now
         if (substr($uri, 0, strlen(self::PREFIX)) !== self::PREFIX) {
-            $uri = self::PREFIX . urlencode($uri);
+            $uri = self::PREFIX . Helper::PLAYLIST_HASH . urlencode($uri);
```

Comment:
Oddly enough the playlist hash is actually required as part of `EnqueuedURI`, which seems to be different when compared to the other sources.